extensions/wikia/Captcha/Factory/Module.class.php

```diff
@@ -48,6 +48,10 @@ public static function verifyCaptchaClass( $captchaClass ) {
 	 * Load Captcha.js on demand when we're getting an instance of a Captcha.
 	 */
 	public static function addCaptchaJS() {
-		\Wikia::addAssetsToOutput( 'captcha_js' );
+		// Make sure we only load FancyCaptcha library once
+		if ( !\F::app()->wg->CaptchaLibraryLoaded ) {
```

Comment:
Ah, good idea @lizlux. I took this approach since I remember we did something similar for the Videos Module (https://github.com/Wikia/app/blob/dev/extensions/wikia/VideosModule/VideosModuleHooks.class.php#L55), but the static variable would probably be a better solution. Let's try and update that the next time we touch this file.